orderbook/src/orderbook.rs

```diff
@@ -195,6 +196,7 @@ impl Orderbook {
             Err(InsertionError::DbError(err)) => return Err(err.into()),
             _ => (),
         }
+        self.solvable_orders.request_update();
```

Comment:
I have a question about this particular line, in combination with L173-182. Specifically, the `can_transfer` check would query the node for balances, and then the `solvable_orders` cache would re-query it when the update here is requested?

Reply:
can_transfer and get_balance are different checks. Can transfer simulates a full transaction from the user to the settlement contract. On the other hand get_balance only does view calls to get state. I believe we added can_transfer as an additional safety measure before we had the bad token detector so not sure if it is still needed.
In the old code it was also the case that can_transfer only happens once on order creation while later we only check get_balance. So this behavior hasn't changed. We could start caching can_transfer too but I not sure it's needed and would do that in a separate PR.

Reply:
Right - for some reason I confused the two. Thanks for clarifying!